### PR TITLE
Update cookie consent library to 3.0.0-rc14.

### PR DESCRIPTION
**Pull request by EreMaijala**
This should fix functionality e.g. with older Safari versions. The less files have been recreated to resemble the scss structure used in the library, and they are no longer so heavily modified for Bootstrap.

### CODE REVIEW

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @EreMaijala! I noticed that you added/changed LESS files here without corresponding changes to SCSS. This led me to open #2910. I think we should merge that and then make appropriate adjustments here to resynchronize the LESS and SCSS files.

Once that is done, I'll do a little hands-on testing, and then can hopefully get this merged!

File: themes/bootstrap3/js/vendor/cookieconsent.umd.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...,e.O=[...e.B]):(e.Z={...e.Z,...a},U([...e.R,...s])),e.ee={...e.Z}})();const i=Oe();if(!await Ce())return!1;if($(null,r=Ie,pe,Se),g.o.M&&be(r,Se),g.t.lazyHtmlGeneration||pe(r,Se),(()=>{const e=g.se,t=g.o;B(e.he,'keydown',(e=>{if('Tab'!==e.key)return;if(!t.k&&!t.T)return;const o=t.W,n=t.K;if(o.length>0){const t=M();e.shiftKey?t!==o[0]&&n.contains(t)||(D(e),z(o[1])):t!==o[1]&&n.contains(t)||(D(e),z(o[0]))}}),!0)})(),n.autoShow&&!i&&Fe(!0),i)return ee(),Z(s.de);n.mode===o&&ee(t.B)}var r},e.setCookieData=e=>{let t,o=e.value,n=e.mode,s=!1;const a=g.o;if('update'===n){a.v=t=Le('data');const e=typeof t==typeof o;if(e&&'object'==typeof t){!t&&(t={});for(let e in o)t[e]!==o[e]&&(t[e]=o[e],s=!0)}else!e&&t||t===o||(t=o,s=!0)}else t=o,s=!0;return s&&(a.v=t,a.p.data=t,xe(!0)),s},e.setLanguage=async(e,t)=>{if(!ve(e))return!1;const o=g.o;return!(e===ye()&&!0!==t||!await Ce(e)||(he(e),o.D&&be(Ie,Se),o.A&&pe(Ie,Se),we(),0))},e.show=Fe,e.showPreferences=je,e.validConsent=Oe,e.validCookie=e=>''!==Te(e,!0)},'object'==typeof exports&&'undefined'!=typeof module?t(exports):'function'==typeof define&&define.amd?define(['exports'],t):t((e='undefined'!=typeof globalThis?globalThis:e||self).CookieConsent={});
```
[the remark this answers is not in the document]
Comment (demiankatz) [reply themes/bootstrap3/js/vendor/cookieconsent.umd.js:7]: @EreMaijala, I assume this is the same false positive that always gets flagged in this library....

[the remark this answers is not in the document]
Comment (EreMaijala) [reply themes/bootstrap3/js/vendor/cookieconsent.umd.js:7]: @demiankatz Yes.

**Comment by EreMaijala** [comment]
@demiankatz Thanks, I never realized it didn't handle deletions!

**Comment by demiankatz** [comment]
> @demiankatz Thanks, I never realized it didn't handle deletions!

Nor did I... and it still doesn't, so we'll need to be careful about that! But at least now it converts files in subdirectories of components.

**Comment by EreMaijala** [comment]
@demiankatz Should be all in sync now.

**Review by demiankatz — APPROVED** [review]
Looks good to me, @EreMaijala! GitHub Actions seems to be taking its time this morning, though, so I can't merge it yet. I'll try again later (though feel free to merge yourself if we have a green checkmark before I get back to this).

**Comment by EreMaijala** [comment]
@demiankatz I suppose the snyk issue is the same as the one github-code-scanning bot found. But the other tests seem to be awol.

**Comment by demiankatz** [comment]
@EreMaijala, I think the whole Snyk service is broken right now and it's interfering with the build process. I tried to log in to look at it and I can't.

**Pull request merged**